src/constants/color.js

```diff
@@ -25,12 +25,14 @@ export const airportOutline = "hsl(250, 41%, 79%)";
 export const airportRunway = "hsl(250, 41%, 79%)";
 export const airportLabel = "hsl(250, 71%, 29%)";
 
+//TODO - rename this variable to "palette"
 export const shields = {
```

Comment:
Maybe we should keep the shield and POI color constants separate, so we have flexibility to make one less intense than the other, for example. I think it’s fine to have `poi` reference `shield`, but `strong_orange` doesn’t need to be part of the overall palette if it’s specific to POIs.

Reply:
Resolved